src/main/java/com/codeborne/selenide/conditions/Or.java

```diff
@@ -18,26 +19,21 @@ public Or(String name, List<Condition> conditions) {
 
   @Override
   public boolean apply(Driver driver, WebElement element) {
-    firstFailedCondition = null;
-
     for (Condition c : conditions) {
       if (c.apply(driver, element)) {
         return true;
       }
-      else if (firstFailedCondition == null) {
-        firstFailedCondition = c;
-      }
     }
     return false;
   }
 
   @Override
   public String actualValue(Driver driver, WebElement element) {
-    return firstFailedCondition == null ? null : firstFailedCondition.actualValue(driver, element);
+    return conditions.stream().map(condition -> condition.actualValue(driver, element)).collect(joining(", "));
   }
 
   @Override
   public String toString() {
-    return firstFailedCondition == null ? super.toString() : firstFailedCondition.toString();
+    return getName() + ": " + conditions.stream().map(Condition::toString).collect(joining(" or "));
```

Comment:
It's better to introduce a new variable to complex stream operation and use string format for a final message

Reply:
done